projects/plugins/jetpack/extensions/blocks/subscriptions/view.scss

```diff
@@ -118,3 +118,12 @@
 		}
 	}
 }
+
+#subscribe-link {
```

Comment:
Would prefer some prefixing here with `jetpack_` to avoid collisions with other plugins/features with generic identifier.

Reply:
Additional ID is already removed. I'm using the one that was already there.